examples/StandardFirmataBLE/StandardFirmataBLE.ino

```diff
@@ -772,14 +768,7 @@ void setup()
   Firmata.attach(START_SYSEX, sysexCallback);
   Firmata.attach(SYSTEM_RESET, systemResetCallback);
 
-  stream.setLocalName(FIRMATA_BLE_LOCAL_NAME);
```

Comment:
I can live with this change, but the reason I used these setters initially was to avoid tightly coupling defines in bleConfig.h with the BLE stream classes. The only reason your change here works is all the code in BLEStream is in the header file, otherwise BLEStream would have been precompiled and the defines in bleConfig.h would not be available due to the weird way the Arduino liker works.

Reply:
Good point.  I was trying to simplify things, but I didn't consider this issue.  I'll add the setters back.